composer.json

```diff
@@ -22,6 +22,7 @@
     "easyframework/collections": "dev-master"
   },
   "require-dev": {
+    "php-amqplib/php-amqplib": "2.6.*",
```

Comment:
`require-dev ` should lists the packages required for developing this package, or running tests.
I would say that we should add both `php-amqplib` and `kafka-php`.